test/bats/helpers/ln.bash

```diff
@@ -86,6 +86,10 @@ lnds_init() {
   retry 10 1 mempool_not_empty
   retry 10 1 no_pending_channels
 
+  # FIXME: we may need some check on the graph or something else
+  # NB: I get randomly a "no route" error otherwise
+  sleep 10
```

Comment:
@vindard I have added a sleep here. AFAIR we had a wait for the network graph to be updated for the e2e test, so we may need the same thing here.